test/alternator/suite.yaml

```diff
@@ -19,5 +19,6 @@ extra_scylla_config_options:
     alternator_timeout_in_ms: 30000,
     alternator_ttl_period_in_seconds: 0.5,
     alternator_streams_time_window_s: 0,
-    tablets_mode_for_new_keyspaces: enabled
+    tablets_mode_for_new_keyspaces: enabled,
+    alternator_max_expression_cache_entries: 0
```

Comment:
This will mean that the expression caching will never get tested by regular tests just using it.